tests/chainer_tests/test_link.py

```diff
@@ -789,6 +801,16 @@ def test_serialize(self):
         mocks['l1'].assert_called_with('x', self.l1.x.data)
         mocks['l2'].assert_called_with('x', self.l2.x.data)
 
+    @unittest.skipUnless(
+        six.PY3, 'Python2.7 has a bug in catch_warnings, so this test is '
+                 'skipped for Python2.7')
```

Comment:
What kind of bug? How about using `testing.assert_warns`?

Reply:
I forgot why I used this. I will try assert_warns

Reply:
Hmm, in Python 2.7, catch_warnings can't count the number of warnings correctly. So the `self.assertEqual(len(w), 2)` fails.

Reply:
Ah, I see. Is this check needed? Warning is printed only once anyway.

Reply:
I think we can remove the warning count checking. I removed them.